### ISSUE TITLE
Fused batch norm can be folded with atrous conv2d 

### ISSUE THREAD

**Issue by liyinhgqw** [tensorflow/tensorflow#13989]
### Describe the problem
As we can fold batch norm with convolution, we should also fold batch norm with atrous convolution, which has not been implemented.

**Comment by smohan10** [comment]
Hi, 

I am working on folding batch norm with atrous convolution for quantization experiments. It seems the Space to Batch -> Conv -> Batch to Space -> Batch Norm -> Activation isn't getting folded correctly. There seems to be a shape incompatibility. Has anyone made progress on this issue?

**Comment by tilakrayal** [comment]
@liyinhgqw,
Respected PR was closed. Could you please check the PR and provide the required information. Thank you!
